data/outfits.txt

```diff
@@ -301,6 +301,15 @@ outfit "Small Bunk Room"
 	unplunderable 1
 	description "Captains who do not care particularly much about the comfort of their passengers or crew can usually find a way to shove a bunk bed in behind the power generator or to hang a few hammocks from the engines as a makeshift way to make space for more people."
 
+outfit "Brig"
+	category "Systems"
+	cost 40000
+	thumbnail "outfit/brig"
+	"mass" 3
+	"outfit space" -3
+	"unplunderable" 1
+	description "With the addition of some reinforced doors, locks, and security systems, any ship's passenger compartment can be turned into a brig, making it suitable for transporting criminals and prisoners... or worse, participating in the highly illegal galactic slave trade. These cells are often found on ships owned by bounty hunters, pirates, and of course, the Republic's interstellar law enforcement."
```

Comment:
That first sentence is really long. I recommend splitting it up, something like this: 

`With the addition of some reinforced doors, locks, and security systems, any ship's passenger compartment can be turned into a brig. This makes it suitable for transporting criminals and prisoners. The most unscrupulous captains use them to participate in the highly illegal galactic slave trade.`

Reply:
You're right, it is really long. How about this?

`With the addition of some reinforced doors, locks, and security systems, any ship's passenger compartment can be turned into a brig. This makes it suitable for transporting criminals and prisoners... or worse, participating in the highly illegal galactic slave trade.`

Reply:
I think supplying a judgement on slave trade is not for the game, i.e. get rid of the "worse"

Reply:
Having an ellipsis in an outfit description also feels a bit off. There's no need for a dramatic pause in a straightforward description. Along with the above, that makes the last part: `This makes it suitable for transporting criminals, prisoners, or slaves.`